CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 0.16.3 (Next)
 =============
 
-* Your contribution here.
+* [#1398](https://github.com/ruby-grape/grape/pull/1398): Option to allow Grape to handle Grape::Exception when rescue :all is set [@mmclead](https://github.com/mmclead)
```

Comment:
Missing a period. Should describe what was added, such as "added rescue_grape_errors to ...".